bundles/org.openhab.binding.jablotron/README.md

```diff
@@ -0,0 +1,120 @@
+
+| thing   | note                                      |
+|---------|-------------------------------------------|
+| bridge  | the bridge to your Jablonet cloud account |
+| JA-80   | the OASIS alarm                           |
+| JA-100  | with the thermometer support              |
+| JA-100F | without the thermometer support           |
+ 
+## Discovery
+
+This binding supports auto discovery. Just manually add a bridge thing and supply login & password to your Jablonet account.
+
+## Binding Configuration
+
+Binding itself doesn't require specific configuration.
+
+## Thing Configuration
+
+| thing                | config parameter name | description                                                                        | type                  |
+|----------------------|-----------------------|------------------------------------------------------------------------------------|-----------------------|
+| bridge               | login                 | the login to your Jablonet account                                                 | mandatory             |
+| bridge               | password              | the password to your Jablonet account                                              | mandatory             |
+| bridge               | refresh               | the refresh time for all alarm warnings (ALARM, TAMPER triggers and SERVICE state) | optional, default=30s |
+| bridge               | lang                  | the language of the alarm texts                                                    | optional, default=en  |
+| JA-80/JA-100/JA-100F | refresh               | the channels refresh period in seconds                                             | optional, default=60s |
+| JA-80/JA-100/JA-100F | serviceId             | the service ID which identifies the alarm                                          | mandatory             |
+| JA-100/JA-100F       | code                  | the master code for controlling sections                                           | optional              |
+
+## Channels
+
+| thing                | channel name     | description                                               |
+|----------------------|------------------|-----------------------------------------------------------|
+| bridge               | N/A              | the bridge does not expose any channels                   |
+| JA-80                | statusA          | the status of the A section                               |
+| JA-80                | statusB          | the status of the AB/B section                            |
+| JA-80                | statusABC        | the status of the ABC section                             |
+| JA-80                | statusPGX        | the status of PGX                                         |
+| JA-80                | statusPGY        | the status of PGY                                         |
+| JA-80                | command          | the channel for sending keyboard codes to the OASIS alarm |
```

Comment:
This sound rather an [Action](https://www.openhab.org/docs/developer/bindings/#actions-bound-to-a-thing) than a Channel.

Reply:
Are you insisting on this? I have to study this a bit, so it will take some time...

Reply:
After some investigation, I would keep it as is, since the action would make the calling more complicated (even it may a good candidate)
now it is possible to simply use this statement in a .sitemap file
```
Switch item=JablotronCode label="Arm" mappings=[1234=" A ", 2345=" B ", 3456="ABC"]
```